tests/karate/src/test/java/logback-test.xml

```diff
@@ -14,7 +14,7 @@
     </encoder>
   </appender>
 
-  <logger name="com.intuit" level="DEBUG"/>
+  <logger name="com.intuit" level="debug"/>
```

Comment:
Just wondering how long it took to figure that this was an issue 😂

Reply:
I was just annoyed by the number of information in the console, so I changed that part. And afterward when resetting it to debug, I wrote it in lowercase, that wasn't an issue